src/chocolatey.tests/infrastructure.app/services/RegistryServiceSpecs.cs

```diff
@@ -47,6 +48,7 @@ protected void reset()
         }
 
         [WindowsOnly]
+		[Platform(Exclude="Unix,Linux,MacOsX")]

```

Comment:
Formatting here looks a bit off.

And why add this when it already has the attribute of `[WindowsOnly]`?

Reply:
Perhaps the attribute needs adjusted?

Reply:
The `[WindowsOnly]` attribute did not work for preventing the test from running on non-windows systems. I looked at the NUnit docs for the correct way to not run the test on non-windows, and it ended up being what I added, see link below.

https://github.com/nunit/docs/wiki/Platform-Attribute

Linux builds still work just fine without the `[WindowsOnly]` attribute, so I could remove it and leave only the `[Platform(Exclude="")]` if you think that would be best.

Reply:
I went ahead and removed the [WindowsOnly] attribute.

Reply:
Sorry, what I was looking for was to fix the WindowsOnly attribute to ensure that it does prevent running on non-Windows systems. I don't want NUnit references required in here. 

Reference `using NUnit.Framework;` at the top of this file.

Reply:
This should do nicely:

change https://github.com/chocolatey/choco/blob/9a432ab44651b2472fe64227183a68cdbc6c9dff/src/chocolatey.tests/TinySpec.cs#L150-L161



to -

~~~c#
    public class WindowsOnlyAttribute : PlatformAttribute
    {
        public WindowsOnlyAttribute()
        {
            Exclude = "Unix,Linux,MacOsX";
        }
    }
~~~

Reply:
WindowsOnly logic has since been broken with the newer compilers - so instead of leaving the broken window it's better to fix the logic.

Reply:
Well, that compiles, but it is still trying(and failing) to run the tests on Linux.

Reply:
So, I could not get it to work, but I did push it so maybe people can look at the Travis build and see if there is an error. I also could have some other error, but I simply could not get it to stop running the tests on Linux.

Reply:
I think maybe it's still missing this (and my bad on that aspect): `public WindowsOnlyAttribute() : base("This is a Windows only test") `

Reply:
https://nunit.org/docs/2.6/platform.html This also highlights some things. I wonder if we could do an Include versus an exclude?

Reply:
Note it is specifically missing MacOSX, but that shouldn't affect Travis.

Reply:
So maybe we can flip to do an include on "Win" and see if that works?

Reply:
> `public WindowsOnlyAttribute() : base("This is a Windows only test")`

I tried adding this, it is still running the tests 

If you want a go at it on windows, try changing the exclude to windows and build on windows then see if it skips the test. 

> So maybe we can flip to do an include on "Win" and see if that works?

Again, still running the tests on Linux.

Reply:
>> `public WindowsOnlyAttribute() : base("This is a Windows only test")`

> I tried adding this, it is still running the tests

Yeah, that was for when it was ignore, we switched it to Platform attribute. Apologies.

>> So maybe we can flip to do an include on "Win" and see if that works?

> Again, still running the tests on Linux.

Darn.